authentication/account/service/user_blackbox_test.go

```diff
@@ -140,6 +143,11 @@ func (s *userServiceBlackboxTestSuite) TestListUsersToNotifyBeforeDeactivation()
 		require.NoError(s.T(), err)
 		require.NotNil(s.T(), identity.DeactivationNotification)
 		assert.True(s.T(), time.Now().Sub(*identity.DeactivationNotification) < time.Second*2)
+		// also verify that the message to send to the user has the correct data
+		assert.Equal(s.T(), identity2.ID.String(), msgToSend.TargetID)
+		expiryDate := time.Now().Add(config.GetUserDeactivationInactivityPeriodDays() - config.GetUserDeactivationInactivityNotificationPeriodDays()).Format("Mon Jan 2")
```

Comment:
I would really like if this logic extracted somewhere, not only for test but for actual code.
```
		expiryDate := time.Now().Add(config.GetUserDeactivationInactivityPeriodDays() - config.GetUserDeactivationInactivityNotificationPeriodDays()).Format("Mon Jan 2")
```
One helper function, which takes required input and return expiryDate (mostly without format) and then caller do the needful.  And then this helper func can be used everywhere.

Reply:
done in 8febf38